projects/onto-search/src/lib/onto-search-facet/range-facet/range-histogram/range-histogram.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="range-histogram-canvas">
```

Comment:
`range-histogram-canvas` this is not the canvas, but the wrapper, right?

Reply:
renamed